### PR TITLE
Implemented film suggestions

### PR DESCRIPTION
**Pull request by Hreewx**
Created film suggestions page with ability to delete and add to review. Created movie page via fetch api with ability to add this current film to suggestions.

### CODE REVIEW

File: src/pages/Movie.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          <h2>About film</h2>
+          <div className={styles.movieData}>
+            <div className={styles.movieRow}>
+              <div className={styles.title}>Production year</div>
+              <div className={styles.value}>{year}</div>
+            </div>
+            <div className={styles.movieRow}>
+              <div className={styles.title}>Slogan</div>
+              <div className={styles.value}>{slogan}</div>
+            </div>
```
Comment (Mikhailosev) [inline src/pages/Movie.jsx]: can be a component, with children props for value

File: src/shared/utils/hooks/reviews/useDeleteReview.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    onSuccess: () => {
+      toast.success("Review was successfully deleted");
+      queryClient.invalidateQueries({
+        queryKey: ["reviews"],
+      });
+    },
+    onError: (err) => toast.error(err.message),
+  });
+  return { isDeleting, deleteReview };
+}
```
Comment (Mikhailosev) [inline src/shared/utils/hooks/reviews/useDeleteReview.js:20]: Why is there no id here?

Comment (Hreewx) [reply src/shared/utils/hooks/reviews/useDeleteReview.js:20]: Снимок экрана 2024-02-10 в 22 15 39

File: src/pages/Movie.jsx (hunk 1 above)
Comment (Hreewx) [reply src/pages/Movie.jsx]: Fixed

**Pull request merged**